### PR TITLE
fix: check whether own inflight count

### PR DESCRIPTION
**Pull request by wu-hanqing**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Review by ilixiaocui — APPROVED** [review]

File: test/integration/client/unstable_chunkserver_exception_test.cpp [hunk 1]
```diff
@@ -223,14 +223,15 @@ class UnstableCSModuleException : public ::testing::Test {
         std::this_thread::sleep_for(std::chrono::seconds(10));
     }
 
-    void TearDown() {
+    static void TearDownTestCase() {
         UnInit();
         ASSERT_EQ(0, cluster->StopCluster());
         // 清理文件夹
         system("rm -rf module_exception_curve_unstable_cs");
+        system("rm -rf ttt");
```
Comment (SeanHai) [inline test/integration/client/unstable_chunkserver_exception_test.cpp:231]: SetUpTestCase中也进行了 system("rm -rf ttt")，是不是只在这 rm就行

File: test/integration/client/unstable_chunkserver_exception_test.cpp [hunk 2]
```diff
@@ -282,12 +283,40 @@ class UnstableCSModuleException : public ::testing::Test {
...
+    static int fd;
+    static std::unique_ptr<CurveCluster> cluster;
+    static std::unordered_map<int, ChunkserverParam> chunkServers;
 };
 
+int UnstableCSModuleException::fd = 0;
+std::unique_ptr<CurveCluster> UnstableCSModuleException::cluster;
+std::unordered_map<int, ChunkserverParam> UnstableCSModuleException::chunkServers; // NOLINT
+
+TEST_F(UnstableCSModuleException, TestCommonReadAndWrite) {
```
Comment (SeanHai) [inline test/integration/client/unstable_chunkserver_exception_test.cpp:299]: 这个测试用例是为了测哪个场景？

File: test/integration/client/unstable_chunkserver_exception_test.cpp (hunk 1 above)
Comment (wu-hanqing) [reply test/integration/client/unstable_chunkserver_exception_test.cpp:231]: 这个影响不大，主要是为了本地跑的时候，能自动清理一下环境。

File: test/integration/client/unstable_chunkserver_exception_test.cpp (hunk 2 above)
Comment (wu-hanqing) [reply test/integration/client/unstable_chunkserver_exception_test.cpp:299]: 这个就是测试CI出现的场景，挂载卷之后读，读会释放inflight导致移除的问题。
单元测试里面不好加测试，所以放到集成测试里面了。

**Review by SeanHai — APPROVED** [review]

**Review by ilixiaocui — APPROVED** [review]

**Pull request merged**